src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -86,21 +86,37 @@ public OutlineAttribute retrieveOutline() {
 			return null;
 		}
 		try {
-			final int x = Integer.parseInt( webData.get( AttributesConfig.X ) );
-			final int y = Integer.parseInt( webData.get( AttributesConfig.Y ) );
-			final int width = Integer.parseInt( webData.get( AttributesConfig.WIDTH ) );
-			final int height = Integer.parseInt( webData.get( AttributesConfig.HEIGHT ) );
+			final int x = toInt( webData.get( AttributesConfig.X ) );
+			final int y = toInt( webData.get( AttributesConfig.Y ) );
+			final int width = toInt( webData.get( AttributesConfig.WIDTH ) );
+			final int height = toInt( webData.get( AttributesConfig.HEIGHT ) );
 			return new OutlineAttribute( new Rectangle( x, y, width, height ) );
 		} catch ( final Exception e ) {
 			logger.error( "Exception retrieving outline: ", e );
 		}
 		return null;
 	}
 
+	private int toInt( final Object value ) {
+		if ( value instanceof Integer ) {
+			return (Integer) value;
+		}
+		if ( value instanceof String ) {
+			return Integer.parseInt( (String) value );
```

Comment:
This will now really throw an `NumberFormatException` when "auto" or not parsable.